js/client-ladder.js

```diff
@@ -79,6 +79,11 @@
 		selectFormat: function (format) {
 			this.curFormat = format;
 			this.update();
+		},
+		refresh: function () {
+			this.$refreshButton = this.$('button[name=refresh]');
+			this.$refreshButton.fadeOut();
```

Comment:
There is no reason for this to be two lines.

Also `fadeOut`? Ugh no that's horrible. Disable it.